client/package.json

```diff
@@ -58,6 +58,7 @@
     "react-addons-css-transition-group": "^15.4.1",
     "react-dom": "^15.3.2",
     "react-redux": "^4.4.5",
+    "react-responsive": "^1.2.5",
```

Comment:
@surtich I'd say there should be only `<QuestionList />` there and the component itself should be responsive.